index.html

```diff
@@ -8,5 +8,6 @@
   </head>
   <body>
     <header class="header"></header>
+    <h1 class="title">Corporate Design Quiz</h1>
```

Comment:
```suggestion
    <header><h1 class="title">Corporate Design Quiz</h1></header>
```
I would suggest you to put your `h1` inside your `header` element, otherwise you could remove the `header`, since he is empty and not used. Also it's not necessary to give the header a class, because you don't style the header for now, or?